docs/getting-started/installation-and-upgrade/resources/bootstrap-password.md

```diff
@@ -6,26 +6,70 @@ title: Setting up the Bootstrap Password
   <link rel="canonical" href="https://ranchermanager.docs.rancher.com/getting-started/installation-and-upgrade/resources/bootstrap-password"/>
 </head>
 
-When Rancher starts for the first time, a password is randomly generated for the first admin user. When the admin first logs in to Rancher, the UI shows commands that can be used to retrieve the bootstrap password. The admin needs to run those commands and log in with the bootstrap password. Then Rancher gives the admin an opportunity to reset the password.
+:::tip
 
-The bootstrap password is randomly generated if it is not set during installation with a variable. For details on how to set the bootstrap password using a variable, see below.
+If you choose not to set a bootstrap password, Rancher randomly generates a bootstrap password for the first admin account.
 
-### Specifying the Bootstrap Password in Helm Installs
+:::
 
-For a Helm install, users can specify the bootstrap password variable by configuring it in the Helm chart values with `.Values.bootstrapPassword`.
+When you install Rancher, you can use a variable to set a bootstrap password for the first admin account.
 
-The password will be stored in a Kubernetes secret. After Rancher is installed, the UI will show instructions for how to retrieve the password using kubectl:
+For details on how to use a variable to set the bootstrap password, see below.
 
+## Password Requirements
+
+The bootstrap password can be any length.
+
+When you reset the first admin account's password after first login, the new password must be at least 12 characters long.
+
+## Specifying the Bootstrap Password
+
+To specify the bootstrap password during initial Rancher installation, run the following commands.
+
+<Tabs>
+<TabItem value="Helm">
+
+Set the following value in the Rancher Helm chart:
+
+```yaml
+.Values.bootstrapPassword
```

Comment:
Line 27 says `run the following commands`, but this isn't a command.

Is this intended to be a snippet of the values.yaml file or a snippet of the Helm install command? In either case, the syntax is incorrect.